cfme/configure/access_control.py

```diff
@@ -262,6 +262,7 @@ def _retrieve_ext_auth_user_groups(self):
 
     def add_group_from_ldap_lookup(self):
         self._retrieve_ldap_user_groups()
+        sel.wait_for_ajax()
```

Comment:
I think mpusater is actually going to end up being the 'author' and I'll be ghost writing so to speak, but yes the ext auth tests need re-written.